src-tauri/src/lib.rs

```diff
@@ -253,7 +253,7 @@ pub fn launch_northstar(
     let host_os = get_host_os();
 
     // Explicitly fail early certain (currently) unsupported install setups
-    if host_os != "windows"
+    if (host_os != "windows" && !matches!(game_install.install_type, InstallType::STEAM))
         || !(matches!(game_install.install_type, InstallType::STEAM)
             || matches!(game_install.install_type, InstallType::ORIGIN)
             || matches!(game_install.install_type, InstallType::UNKNOWN))
```

Comment:
This is a mess but I don't know how this could be improved.

Reply:
Imma make a new issue to maybe figure out control flow for launching Northstar ^^

Reply:
Got a potential flow chart on how to structure the control flow in https://github.com/R2NorthstarTools/FlightCore/issues/179#issuecomment-1426895213

Feel free to leave any comments/concerns/ideas/etc there.

Question is, do we wanna change control flow as part of this PR or figure it out in a separate one? ^^

Reply:
Marking as _resolved_ as no longer relevant for this PR ^^